sdk/src/types/block/output/feature/mod.rs

```diff
@@ -264,6 +290,15 @@ impl Features {
             None
         }
     }
+
+    /// Gets a reference to a [`StakingFeature`], if any.
+    pub fn staking(&self) -> Option<&StakingFeature> {
+        if let Some(Feature::Staking(staking)) = self.get(StakingFeature::KIND) {
+            Some(staking)
+        } else {
+            None
+        }
```

Comment:
map?

Reply:
What do you want to map?

Reply:
`self.get(StakingFeature::KIND).map(|f| f.as_staking()))`

Reply:
Yeah I guess that could work 👍🏻

Reply:
I'll do it on develop, rebase and do it here as well

Reply:
awesome :+1:

Reply:
Done